blazor/dropdown-list/filtering.md

```diff
@@ -7,103 +7,116 @@ control: DropDown List
+
+## Remote data
+
+In the below code, demonstrated the filtering functionality with [ODataAdaptor](https://blazor.syncfusion.com/documentation/data/adaptors#odata-adaptor) in the DropDownList component with help of [Query](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Data.Query.html) property.
+
+{% highlight cshtml %}
+
+{% include_relative code-snippet/filtering/remote-data.razor %}
+
+{% endhighlight %}
+
+![Blazor DropdownList with Remote Data filtering](./images/filtering/blazor_dropdown_remote-data.png)
+
+## Filter type
+
+You can use [FilterType](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfDropDownBase-1.html#Syncfusion_Blazor_DropDowns_SfDropDownBase_1_FilterType) property to specify on which filter type, the component needs to be considered on search action. The available `FilterType` and its supported data types are:
+
+FilterType     | Description
+------------ | -------------
+  StartsWith       | Checks whether a value begins with the specified value.
+  EndsWith     | Checks whether a value ends with specified value.
+  Contains      | Checks whether a value contains with specified value.
+
+In the following example, `EndsWith` filter type has been mapped to the `FilterType` property.
+
+{% highlight cshtml %}
+
+{% include_relative code-snippet/filtering/filter-type.razor %}
+
+{% endhighlight %}
+
+![Blazor DropdownList with Filter Type](./images/filtering/blazor_dropdown_filter-type.png)
+
+## Case sensitive filtering
+
+The Data items can be filtered either with or without case sensitivity using the [DataManager](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Data.SfDataManager.html). This can be done by passing the fourth optional parameter [IgnoreCase](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Data.WhereFilter.html#Syncfusion_Blazor_Data_WhereFilter_IgnoreCase) of the [Where clause](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Data.Query.html#Syncfusion_Blazor_Data_Query_Where_Syncfusion_Blazor_Data_WhereFilter_).
+
+The following example shows how to perform case-sensitive filter.
+
+{% highlight cshtml %}
+
+{% include_relative code-snippet/filtering/case-sentitive.razor %}
+
+{% endhighlight %}
+
+## Filter textbox placeholder 
+
+You can use [FilterBarPlaceholder](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfDropDownList-2.html#Syncfusion_Blazor_DropDowns_SfDropDownList_2_FilterBarPlaceholder) to accept the value to be displayed as a watermark text on the filter bar.
+
+{% highlight cshtml %}
+
+{% include_relative code-snippet/filtering/filter-textbox-placeholder.razor %}
+
+{% endhighlight %}
+
+![Blazor DropdownList with Filter Textbox Placeholder](./images/filtering/blazor_dropdown_filter-textbox-placeholder.png)
+
+## Custom filtering
+
+DropDownList component filter queries can be customized. You can also use your own filter libraries to filter data like Fuzzy search.
```

Comment:
- How we achieve this using Filter event right? we need to explain this in words and like the Event API. 
- Use the below content - DropDownList component filter queries can be customized using [Filter] event. You can also filter the text in multiple columns in the data source.